src/main/java/games/strategy/engine/lobby/server/db/Database.java

```diff
@@ -42,38 +35,9 @@
 public class Database {
   private static final Logger logger = Logger.getLogger(Database.class.getName());
   private static final Object dbSetupLock = new Object();
-  private static boolean isDbSetup = false;
+  private static final boolean isDbSetup = false;
```

Comment:
It looks like this field is no longer used.

Reply:
good catch, though can one up ya a bit better on this  : ) 
The DB table creation is done by travis, we no longer need this bit for derby tables, can remove this var and the other supporting ones nearby.